packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -355,7 +340,7 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
       subItem: true,
     }, {
       id: 'about',
-      title: 'About the Forum',
+      title: 'How to use the Forum',
```

Comment:
Requested by Sharang in [slack](https://cea-core.slack.com/archives/C038J512SD8/p1666880898598739?thread_ts=1666569460.680499&cid=C038J512SD8)